src/main/resources/application.properties

```diff
@@ -48,3 +48,36 @@ app.table.detectedLicenseSchema=lpvs
 app.table.diffFileName=code_licenses
 app.table.pullRequestsName=lpvs_pull_requests
 app.table.queueName=queue
+
+# Login Configuration
+#Google
+spring.security.oauth2.client.registration.google.client-id={YOUR_GOOGLE_CLIENT_ID}
```

Comment:
@o-kopysov , need your opinion here, do we follow same rule for whole `apllication.propeties` file, where 
required fields is left empty?

Reply:
An error happens in LPVS if `client-id` is empty. 
I'd like to put mock value to them so that LPVS can run without login service.